src/apis/api.js

```diff
@@ -0,0 +1,27 @@
+// https://bootcamp-api.codeit.kr/api/sample/user
+
+const baseUrl = 'https://bootcamp-api.codeit.kr/api';
```

Comment:
P1:
일반적으로 상수는 대문자로 명명합니다.

```suggestion
const BASE_URL = 'https://bootcamp-api.codeit.kr/api';
```